InternetHospital.BusinessLogic/Services/DoctorService.cs

```diff
@@ -104,5 +113,52 @@ private IQueryable<DoctorModel> PaginationHelper(IQueryable<Doctor> doctors, int
             return doctorsModel;
         }
 
+        public DoctorProfileModel GetDoctorProfile(int userId)
+        {
+            var user = _context.Users
+                .Include(u => u.Doctor)
+                .Include(u => u.Doctor.Specialization)
+                .FirstOrDefault(u => u.Id == userId);
+
+            var doctorModel = new DoctorProfileModel(); 
+                
+            Mapper.Map(user, doctorModel, opt => opt.AfterMap((u, dm) =>
+            {
+                dm.Address = u.Doctor.Address;
+                dm.Specialization = u.Doctor.Specialization.Name;
+            }));
+            
+            return doctorModel;
+        }
+
+        public async Task<bool> UpdateDoctorInfo(DoctorProfileModel doctorModel, int userId, IFormFileCollection passport, IFormFileCollection diploma, IFormFileCollection license)
+        {
+            var addedTime = DateTime.Now;
+            var result = true;
+            var doctor = _context.Users
+                .Include(u => u.Doctor)
+                .FirstOrDefault(u => u.Id == userId);
+            if (doctor == null)
+            {
+                result = false;
+                return result;
```

Comment:
I think "result" variable is unnecessary. You can return "false" here and "true" at the end.